frame/utility/src/lib.rs

```diff
@@ -110,13 +110,33 @@ pub mod pallet {
 		BatchCompleted,
 	}
 
+	#[pallet::extra_constants]
+	impl<T: Config> Pallet<T> {
+		/// The limit on the number of batched calls.
+		fn batched_calls_limit() -> u32 {
+			let allocator_limit = 33554432; // 32MiB
```

Comment:
should be available now thanks to other pr?

Reply:
yes I'll update once https://github.com/paritytech/substrate/pull/9393 is merged

Reply:
fixed